### PR TITLE
Made changes in the profile section

### PR DESCRIPTION
**Pull request by shruti8302**
Screenshot (428)

### CODE REVIEW

**Comment by narayan954** [comment]
Also, please mention the issue number

**Comment by shruti8302** [comment]
Issue No. - #385

**Comment by shruti8302** [comment]
> Hi @shruti8302 can you please resolve the conflicts? Sorry for the inconvenience

I did.

File: src/pages/Profile.jsx [hunk 1]
```diff
@@ -6,11 +6,12 @@ import {
   Typography,
   useMediaQuery,
 } from "@mui/material";
+import { FaBlackTie, FaUserCircle } from "react-icons/fa";
 import { auth, db, storage } from "../lib/firebase";
 import { useEffect, useState } from "react";
 import { useLocation, useNavigate } from "react-router-dom";
 
-import { FaUserCircle } from "react-icons/fa";
+import { NoEncryptionTwoTone } from "@mui/icons-material";
```
Comment (narayan954) [inline src/pages/Profile.jsx:14]: icon is not used

**Review by narayan954 — CHANGES_REQUESTED** [review]
Hi @shruti8302 , it's coming out to be like this :/ 
Can you please fix that?
image

File: src/pages/Profile.jsx [hunk 2]
```diff
@@ -6,11 +6,12 @@ import {
   Typography,
   useMediaQuery,
 } from "@mui/material";
+import { FaBlackTie, FaUserCircle } from "react-icons/fa";
```
[the remark this answers is not in the document]
Comment (shruti8302) [reply src/pages/Profile.jsx]: I have removed fablacktie.

File: src/pages/Profile.jsx (hunk 1 above)
Comment (shruti8302) [reply src/pages/Profile.jsx]: I have removed 
import { NoEncryptionTwoTone } from "@mui/icons-material";

**Comment by shruti8302** [comment]
> I have added a commit to remove the syntax error due to merge conflict.. please try git pull before making changes

Okay

**Comment by shruti8302** [comment]
I have made the required changes.
Screenshot (439)
Screenshot (440)
Screenshot (441)

**Review by shruti8302 — COMMENTED** [review]
I have made the required changes.

**Comment by narayan954** [comment]
> I have made the required changes.

Cool, reviewing them

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Comment by narayan954** [comment]
Thanks for your contribution! :)

**Pull request merged**